ddcommon/tests/cgroup.multiple_controllers

```diff
@@ -0,0 +1,10 @@
+10:blkio:/user.slice/user-0.slice/session-14.scope
```

Comment:
_Nitpick:_ I would prefer that these files were in `resource/test` instead of the standard integration test directory.

Reply:
There is no rust/cargo convention for resources, but there is a convention for integration tests being in the `tests` directory. I’m not sure how we want to organize this, so this is fine for now.